2_munge/src/aggregate_inputs.R

```diff
@@ -1,3 +1,38 @@
-aggregate_inputs <- function() {
+aggregate_inputs <- function(ind_file, raw_ind_file, remake_file, sites_yml, gd_config) {
 
+  # aggregating NWIS data to a daily scale
+  sites = yaml::yaml.load_file(sc_retrieve(sites_yml,remake_file = remake_file))
+
+  input_raw = readRDS(sc_retrieve(raw_ind_file, remake_file = remake_file))
+
+  agg_sensor = lapply(sites, function(site){
+    agg_out = input_raw$nitrate_sensor %>%
+      dplyr::filter(site_no == site) %>% # stats package overrides dplyr filter()
+      mutate(date = as.Date(dateTime)) %>%
+      group_by(date) %>%
+      summarise(X_99133_000000_mean = mean(X_99133_00000)) %>%
+      mutate(site_no = site, tz_cd = 'UTC')
+  }) %>% bind_rows() %>% as_data_frame()
+
+  agg_flow = lapply(sites, function(site){
+    agg_out = input_raw$flow %>%
+      dplyr::filter(site_no == site) %>% # stats package overrides dplyr filter()
+      mutate(date = as.Date(dateTime)) %>%
+      group_by(date) %>%
+      summarise(X_00060_00000_mean = mean(X_00060_00000)) %>%
```

Comment:
it might be too much of a pain to go back and change this, but for future reference, `dataRetrieval::renameNWISColumns()` can help make these columns look nicer when you retrieve them

Reply:
ah yeah that is helpful. Thanks!